CMake/packages/Findlikwid.cmake

```diff
@@ -0,0 +1,18 @@
+# from git@github.com:PolyJIT/polli.git
```

Comment:
This CMake module doesn't follow the _standards_ [requirements](https://cmake.org/cmake/help/latest/manual/cmake-developer.7.html#find-modules)

This module is also missing documentation in prelude that details the list of variables defined. I also recommend using the [find_package_handle_standard_args ](https://cmake.org/cmake/help/latest/module/FindPackageHandleStandardArgs.html) CMake function to take care of most things.

You can take inspiration from supported modules in CMake install directory.

Reply:
could you please take a look at 3f280985ac66b40f152a558dcee97d374c40be42?